reference/features/search_parameters.md

```diff
@@ -299,25 +299,27 @@ If `attributesToCrop` is not configured, `cropLength` has no effect on the retur
 **Expected value**: an array of <clientGlossary word="attribute" label="attributes" /> or `["*"]`
 **Default value**: `null`
 
-Highlights matching query terms in the given attributes. When this parameter is set, the `_formatted` object is added to the response for each document, within which you can find the highlighted text.
+Highlights matching query terms in the specified attributes by enclosing them in `<em>` tags. `attributesToHighlight` highlights matches in fields containing strings, numeric values, arrays, and objects.
```

Comment:
"Numeric value" is inconsistent IMO since in a MeiliSearch context these are all "values".

```suggestion
Highlights matching query terms in the specified attributes by enclosing them in `<em>` tags. `attributesToHighlight` only works on values of the following types: string, number, array, object.
```